nemo/collections/asr/modules/conv_asr.py

```diff
@@ -174,6 +174,8 @@ def __init__(
             )
             feat_in = lcfg['filters']
 
+        self._feat_out = feat_in
+
```

Comment:
What is this line for? Is it needed?

Reply:
It is to enable a feature that we do not need to set the feat_in of the decoder directly in the config file. Now when feat_in of the decoder is not set, it tries to set the feat_in of decoder automatically to feat_out of decoder.

Reply:
Where is this done?

Reply:
Here:  https://github.com/NVIDIA/NeMo/blob/7dabb7844dbce1128c9bceb308718c69c4ac91f1/nemo/collections/asr/models/ctc_models.py#L103